encoding/protobuf/protobuf_test.go

```diff
@@ -38,25 +37,6 @@ func TestDecodeError(t *testing.T) {
 	assert.Error(t, err)
 }
 
-func TestProtobuf(t *testing.T) {
```

Comment:
This test is no more needed?

Reply:
Thsi test was removed because it tried to increased the coverage by testing methods of the protobuf message.
some of them have been removed so we decided to remove such an obvious test. We could return it but it is implicitly testing in prior tests.